src/Components/Profile/PreferenceSelector.tsx

```diff
@@ -57,7 +57,7 @@ export const PreferenceSelector = () => {
       <div className={`${collapsed? "collapsed " : ""}profile-render`}>
         <div className="hstack">
           <div className="dropdown-label">
-            Preferred Number of Lab Sections:  
+            Preferred Number of Weekly Lab Hours:
           </div>
           <input id={input_id} type="number" placeholder={user.user? `${user.user.desired_number_assignments}` : '2'} style={{margin: '0 5px'}}/>
```

Comment:
Has your PR been merged @benbeauchamp7? Looks like when I checked out `main` this line was reverted somehow. I'm not sure what you did to make it work, but I'll leave this suggestion so you can autocommit if you want.
```suggestion
          <input id={input_id} type="number" value={user.user? `${user.user.desired_number_assignments}` : '2'} style={{margin: '0 5px'}}/>
```